cogs5e/sheets/dicecloud.py

```diff
@@ -45,8 +45,8 @@
     "sorceryPoints": "long",
     "superiorityDice": "short",
 }
-API_BASE = "https://dicecloud.com/character/"
-DICECLOUD_URL_RE = re.compile(r"(?:https?://)?dicecloud\.com/character/([\d\w]+)/?")
+API_BASE = "https://v1.dicecloud.com/character/"
+DICECLOUD_URL_RE = re.compile(r"(?:https?://)?(?:v1\.)?dicecloud\.com/character/([\d\w]+)/?")
```

Comment:
Should probably change this to ONLY use v1 :p
```suggestion
DICECLOUD_URL_RE = re.compile(r"(?:https?://)?v1\.dicecloud\.com/character/([\d\w]+)/?")
```

Reply:
Oh yeah, I guess now we should, this is just from your commit, I forgot to check it